src/getCustomLibp2p.ts

```diff
@@ -17,12 +17,12 @@ import { webSockets } from '@libp2p/websockets'
 import { ipnsSelector } from 'ipns/selector'
 import { ipnsValidator } from 'ipns/validator'
 import { createLibp2p as create, type Libp2pOptions } from 'libp2p'
-import { DELEGATED_ROUTING_V1_HOST, USE_DELEGATED_ROUTING, USE_LIBP2P } from './constants.js'
+import { DELEGATED_ROUTING_V1_HOST, USE_DELEGATED_ROUTING } from './constants.js'
 import type { Libp2p, PubSub, ServiceMap } from '@libp2p/interface'
 import type { HeliaInit } from 'helia'
 
 interface HeliaGatewayLibp2pServices extends Record<string, unknown> {
-  dht: DualKadDHT
+  dht: KadDHT
   delegatedRouting: unknown
   pubsub: PubSub
   relay: CircuitRelayService
```

Comment:
Why do we need the circuit relay service?  We don't want to be a relay server so it can be removed?

Reply:
removed